classes/book.rb

```diff
@@ -0,0 +1,11 @@
+# Define class Book
+class Book
+  attr_reader :rentals
```

Comment:
- [ ]  Good job for creating the attributes for Book class. One thing to be improved here is that Rental attribute should have both `setter `and `getter`.  When you use `attr_reader `on Rental , you limit it to be changed or updated . fix it for your app to work properly.